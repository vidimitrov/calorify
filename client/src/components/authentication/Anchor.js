import styled from 'styled-components';
import { Link } from '@reach/router';

const Anchor = styled(Link)`
  color: #e5c769;
  display:inline-block;
  text-decoration: none;
  font-weight: 400;
`;

export default Anchor;
