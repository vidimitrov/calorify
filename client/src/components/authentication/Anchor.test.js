import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Anchor from './Anchor';

it('Authentication/Anchor component renders correctly', () => {
  const tree = renderer.create(<Anchor to="/auth/login">Login</Anchor>).toJSON();
  expect(tree).toMatchSnapshot();
});
