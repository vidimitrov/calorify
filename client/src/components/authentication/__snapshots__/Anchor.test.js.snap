// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication/Anchor component renders correctly 1`] = `
.c0 {
  color: #e5c769;
  display: inline-block;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 400;
}

<a
  className="c0"
  href="/auth/login"
  onClick={[Function]}
>
  Login
</a>
`;
